build.rs

```diff
@@ -124,8 +131,19 @@ fn main() {
         println!("cargo:rerun-if-changed=proto/google/rpc/status.proto");
         println!("cargo:rerun-if-changed=proto/vector.proto");
 
+        // Create and store the "file descriptor set" from the compiled Protocol Buffers packages.
+        // This allows us to use runtime reflection to manually build Protocol Buffers payloads
+        // in a type-safe way, which is necessary for incrementally building certain payloads, like
+        // the ones generated in the `datadog_metrics` sink.
+        let protobuf_fds_path = std::env::var("OUT_DIR")
+            .map(PathBuf::from)
+            .map(|path| path.join("protobuf-fds.bin"))
+            .expect("OUT_DIR environment variable not set");
```

Comment:
I don't think it should be a const, no, since it's only used once in the build script, and we can't share it to use it in `src/proto/mod.rs`, since we need a string literal for `include_bytes!`.